core/tests/protractor_utils/workflow.js

```diff
@@ -29,8 +30,9 @@ var createExploration = function() {
 
 // Creates a new exploration and wait for the exploration tutorial to start.
 var createExplorationAndStartTutorial = function() {
-  browser.get(general.LIBRARY_URL_SUFFIX);
-  element(by.css('.protractor-test-create-activity')).click();
+  libraryPage = new LibraryPage.LibraryPage();
```

Comment:
Put this in a beforeEach?

Reply:
I'm not sure how that would work in this utility file since there are no test blocks in here. The function is only exported and used by other files.

Reply:
hmm I see... looks like this entire function belongs in its own separate Page Object, so we'll give this a proper home later. Thanks!